.eslintrc.js

```diff
@@ -252,7 +252,7 @@ module.exports = {
       },
     },
     {
-      files: ['packages/{rum,logs,rum-slim}/src/index.ts', 'packages/rum/src/internal.ts'],
+      files: ['packages/{rum,logs,rum-slim}/src/index.ts', 'packages/rum/src/{internal,internalSynthetics}.ts'],
```

Comment:
I'm wondering about `internalSynthetics.ts`. Shouldn't it be named like `rum.synthetics.entry.ts` since it provide the same ability as `rum.entry.ts` but for synthetics?

Reply:
Good point. While it does about the same job as `rum.entry.ts`, it is a "package entry point" similar to `index.ts`, that's why I think it should stay at the same level as `index.ts`. But maybe we could improve things a bit and have a "entries" folder, like so:

```
 packages/rum/entries/
    main.ts // combination of rum.entry.ts and index.ts
    internal.ts
    internalSynthetics.ts
```
WDYT? cc @bcaudan

Reply:
Yes, I think it could be nice.